core/pyproject.toml

```diff
@@ -27,6 +27,7 @@ dependencies = [
     "langchain-openai==0.1.7",
     "langchain-anthropic==0.1.23",
     "langchain-google-genai==1.0.8",
+    "langchain-google-genai[images]",
```

Comment:
why do you need it?

Reply:
Without this package the Google LLMs do not support data uri images, the api gives an error saying no mimetype is specified

https://pypi.org/project/langchain-google-genai/